### PR TITLE
EOS-20027: Added default firewall config file before field prepare co…

### PR DESCRIPTION
**Pull request by tanujashinde0405**
…mmands

Signed-off-by: Tanuja Shinde <tanuja.shinde@seagate.com>

### CODE REVIEW

**Review by sradhanand-pati — APPROVED** [review]

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: srv/components/system/firewall/files/firewall_config.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - http
+      - https
+      - ftp
+      - ntp
+      - salt-master
+      - ssh
+      - glusterfs
+    ports:
+      csm:
+        - 28100/tcp
```
Comment (nitin-seagate) [inline srv/components/system/firewall/files/firewall_config.yaml:24]: Add new line to file...

**Review by nitin-seagate — APPROVED** [review]
LGTM

File: srv/components/system/firewall/files/firewall_config.yaml (hunk 1 above)
Comment (tanujashinde0405) [reply srv/components/system/firewall/files/firewall_config.yaml]: Done

**Review by 83bhp — APPROVED** [review]

**Pull request merged**